packages/dbgpt-core/src/dbgpt/util/cli/_module.py

```
@@ -37,7 +44,10 @@ def _scan_awel_flow(modules: Optional[List[str]] = None):
             class_filter=lambda cls: _is_flow_operator(cls) or _is_flow_resource(cls),
             skip_files=["test_*.py", "*_test.py"],
         )
-        scanner.scan_and_register(config)
+        try:
+            scanner.scan_and_register(config)
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)